.gitlab-ci.yml

```diff
@@ -153,79 +153,66 @@ TF_Service::Wrapper::build:
     expire_in: 2 weeks
     when: always
 
-TF_Service::dotNet::Windows::build:
+.Base_TF_Service_dotNet:
   extends: .Always_Build
+  variables:
+      DEFAULT_RELEASE_SUFFIX: ""
+      BRIGHTSIGN: ""
+      ARTIFACT_NAME: ""
+      SIGN_BINARY: ""
+      RUNTIME: ""
   tags:
     - docker-windows
   image: registry.ultrahaptics.com/ooh/infrastructure/windows-dotnet-toolchain:latest
   script:
     - if ($CI_COMMIT_REF_NAME -Match 'release/.*') {
-        $TOUCHFREE_SUFFIX = ''
+        $TOUCHFREE_SUFFIX = $DEFAULT_RELEASE_SUFFIX
       } else {
         $TOUCHFREE_SUFFIX = 'dev+' + ${env:CI_COMMIT_SHORT_SHA}
       }
 
-    - dotnet publish -p:VersionPrefix=$TOUCHFREE_VERSION -p:VersionSuffix=$TOUCHFREE_SUFFIX -c Release -r win-x64 "$CI_PROJECT_DIR/TF_Service_dotNet/TouchFree/TouchFreeLib.csproj"
-    - dotnet publish -p:VersionPrefix=$TOUCHFREE_VERSION -p:VersionSuffix=$TOUCHFREE_SUFFIX -c Release -r win-x64 "$CI_PROJECT_DIR/TF_Service_dotNet/TouchFree_Service/TouchFree_Service.csproj"
+    - dotnet publish -p:BRIGHTSIGN=$BRIGHTSIGN -p:VersionPrefix=$TOUCHFREE_VERSION -p:VersionSuffix=$TOUCHFREE_SUFFIX -c Release -r $RUNTIME "$CI_PROJECT_DIR/TF_Service_dotNet/TouchFree/TouchFreeLib.csproj"
```

Comment:
I can't find what the -p option does but I assume it's a variable definition. Have you any docs on this?